examples/html/index.html

```diff
@@ -21,6 +21,15 @@
     <h1>UMA Auth Client html demo</h1>
     <p>Click the UMA Connect button to get started.</p>
     <script src="../../packages/uma-auth-client/dist/uma-auth-client-web-components.umd.cjs"></script>
-    <uma-connect-button></uma-connect-button>
+    <uma-connect-button
+      nostr-pubkey="npub1scmpzl2ehnrtnhu289d9rfrwprau9z6ka0pmuhz6czj2ae5rpuhs2l4j9d"
+      nostr-relay="wss://nos.lol"
+      redirect-uri="http://localhost:3001"
+      required-commands="pay_invoice,get_balance"
```

Comment:
Should this be a list of commands instead from a typing standpoint or does it always have to be a string with webcomponents?

Reply:
has to be a string in this example bc this is just raw html and not jsx